### PR TITLE
Issue #7230 fixed

### LINKED ISSUE #7230: sorting should be work only after clicking on the tax of table header.
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

Master

### Issue Description

shorting should work only after clicking on the tax of the table header.

### Preconditions

**1.** Admin should be login 
**2.** Admin should be on the products page inside the catalog section
**3.** Some products should be added by the admin

### Steps to reproduce

**1.** Go to the 'admin' panel
**2.** Click on 'products' inside the catalog.
**3.** Scroll down to 'click on slightly right of the SKU' on the table header section.
**4.** See the error

### Expected Result

shorting should work only after clicking on the tax on the table header.

### Actual Result

shorting is working if the admin clicks slightly right/left of the header tax.

### Screenshots

k

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Sanjaybhattwebkul**
#7230
> Add span tag inside tbody/th

### CODE REVIEW

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-table.vue [hunk 1]
```diff
@@ -112,25 +112,30 @@
...
-                            : ''
-                    "
-                    v-on:click="
-                        typeof column.sortable !== 'undefined' &&
-                        column.sortable
-                            ? sortCollection(column.index)
-                            : {}
-                    "
-                ></th>
+
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-table.vue:119]: Remove the new line.

File: packages/Webkul/Ui/src/Resources/assets/sass/app.scss [hunk 2]
```diff
@@ -613,7 +613,7 @@ h5 {
             background: #f8f9fa;
             color: $font-color;
 
-            &.sortable {
+            span{
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/sass/app.scss:616]: Add one space after the span.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-table.vue [hunk 3]
```diff
@@ -20,25 +20,29 @@
...
-                        typeof column.sortable !== 'undefined' &&
-                        column.sortable
-                            ? sortCollection(column.index)
-                            : {}
-                    "
-                ></th>
+                >
+                    <span
+                        v-text="column.label"
+                        class="grid_head"
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-table.vue:30]: Class name should be `kebab-case`.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by amitkumarqa** [comment]
@Sanjaybhattwebkul,

Resolve the conflict.

**Pull request closed without merging**